Jenkinsfile

```diff
@@ -46,6 +46,11 @@ pipeline {
             stage('KEVM help')      { steps { sh './kevm help'                                                    } }
           }
         }
+        stage('Test kevm_pyk') {
+          failFast true
+          options { timeout(time: 60, unit: 'SECONDS') }
+          steps { sh 'make test-kevm-pyk' }
+        }
```

Comment:
This should be a sub-stage of `Test Interactive` I would think, unless you needed it to run after that for a reason? It can just be one line, after the `make test-node` line.